ISM43362Interface.cpp

```diff
@@ -85,7 +85,7 @@ int ISM43362Interface::connect()
 
     _ism.setTimeout(ISM43362_CONNECT_TIMEOUT);
 
-    if (!_ism.connect(ap_ssid, ap_pass)) {
+    if (!_ism.connect(ap_ssid, ap_pass, (uint32_t)ap_sec)) {
```

Comment:
here don't we need to map between ap_sec mbed types definition to ISM ism_security_t new types ?

Reply:
done in a cleaner way. Thanks